src/sentry/testutils/hybrid_cloud.py

```diff
@@ -199,19 +191,19 @@ def new_atomic_on_commit(func, using=None):
         _old_transaction_on_commit(func, using)
         maybe_flush_commit_hooks(transaction.get_connection(using))
 
+    # django tests start inside two transactions
+    if is_django_test_case:
+        for db_name in settings.DATABASES:
+            simulated_transaction_watermarks.state[db_name] = 1
```

Comment:
Should we be using `connection.savepoint_ids` here to get the watermark depth?

Reply:
Mmmmmmmmm you're right, k.